### ISSUE TITLE
Syncing watch history of incorrect user

### ISSUE THREAD

**Issue by pterisaur** [Taxel/PlexTraktSync#1339]
### Confirmation

- [X] I have read the [README.md](https://github.com/Taxel/PlexTraktSync#readme) on the project homepage
- [X] I have checked if identical issue already exists
- [X] I have tried downgrading to find version that can be used as a workaround

### The problem

My partner's instance of PlexTraktSync (on his own computer) is randomly syncing episodes from the Plex server owner's (my own) watch history to his account, along with his own watch history. What's strange is this is completely sporadic — not all items from my watch history are syncing to his account, just a handful of seasons from random shows he's never watched. 

Sync watched_status: true in both directions, and watch username_filter: true (though I think what's happening here is a watched status sync, not scrobbling).

### Error trace / logs

```text
##### Log file was over 4GB, so I've pasted a section containing an incorrect episode ("Emily in Paris"). #####


2023-01-02 09:11:38,329 DEBUG[trakt.core]:get: https://api-v2launch.trakt.tv/search/tmdb/4366?type=show
2023-01-02 09:11:38,329 DEBUG[trakt.core]:method, url :: get, https://api-v2launch.trakt.tv/search/tmdb/4366?type=show
2023-01-02 09:11:38,330 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,330 DEBUG[trakt.core]:RESPONSE [get] (https://api-v2launch.trakt.tv/search/tmdb/4366?type=show): request: GET https://api-v2launch.trakt.tv/search/tmdb/4366?type=show, response: 200 (191 bytes), created: 2022-10-25 14:07:11 MDT, expires: N/A (fresh)
2023-01-02 09:11:38,330 DEBUG[plexapi]:GET https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218949/allLeaves
2023-01-02 09:11:38,331 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,332 DEBUG[requests_cache.policy.actions]:URL https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218949/allLeaves matched pattern "*/library/metadata/*/allLeaves": 0
2023-01-02 09:11:38,340 DEBUG[urllib3.connectionpool]:https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400 "GET /library/metadata/218949/allLeaves HTTP/1.1" 200 941
2023-01-02 09:11:38,341 DEBUG[requests_cache.session]:Pre-cache checks for response from https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218949/allLeaves: {'disabled cache': False, 'disabled method': False, 'disabled status': False, 'disabled by filter': False, 'disabled by headers or expiration params': True}
2023-01-02 09:11:38,341 DEBUG[requests_cache.session]:Skipping cache write for URL: https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218949/allLeaves
2023-01-02 09:11:38,342 DEBUG[plexapi]:Reloading Episode 'Show 1' for attr 'guids'
2023-01-02 09:11:38,343 DEBUG[plexapi]:GET https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218951?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1
2023-01-02 09:11:38,344 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,344 DEBUG[requests_cache.policy.actions]:URL https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218951?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1 matched pattern "*/library/metadata/*?*include*": 0
2023-01-02 09:11:38,359 DEBUG[urllib3.connectionpool]:https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400 "GET /library/metadata/218951?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1 HTTP/1.1" 200 1264
2023-01-02 09:11:38,359 DEBUG[requests_cache.session]:Pre-cache checks for response from https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218951?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1: {'disabled cache': False, 'disabled method': False, 'disabled status': False, 'disabled by filter': False, 'disabled by headers or expiration params': True}
2023-01-02 09:11:38,359 DEBUG[requests_cache.session]:Skipping cache write for URL: https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218951?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1
2023-01-02 09:11:38,361 DEBUG[trakt.core]:get: https://api-v2launch.trakt.tv/shows/4342/seasons?extended=episodes
2023-01-02 09:11:38,361 DEBUG[trakt.core]:method, url :: get, https://api-v2launch.trakt.tv/shows/4342/seasons?extended=episodes
2023-01-02 09:11:38,363 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,363 DEBUG[requests_cache.policy.actions]:URL https://api-v2launch.trakt.tv/shows/4342/seasons?extended=episodes matched pattern "*.trakt.tv/shows/*/seasons": 0
2023-01-02 09:11:38,401 DEBUG[urllib3.connectionpool]:https://api-v2launch.trakt.tv:443 "GET /shows/4342/seasons?extended=episodes HTTP/1.1" 200 None
2023-01-02 09:11:38,402 DEBUG[requests_cache.session]:Pre-cache checks for response from https://api-v2launch.trakt.tv/shows/4342/seasons?extended=episodes: {'disabled cache': False, 'disabled method': False, 'disabled status': False, 'disabled by filter': False, 'disabled by headers or expiration params': True}
2023-01-02 09:11:38,402 DEBUG[requests_cache.session]:Skipping cache write for URL: https://api-v2launch.trakt.tv/shows/4342/seasons?extended=episodes
2023-01-02 09:11:38,402 DEBUG[trakt.core]:RESPONSE [get] (https://api-v2launch.trakt.tv/shows/4342/seasons?extended=episodes): <Response [200]>
2023-01-02 09:11:38,408 DEBUG[PlexTraktSync]:The Electric Company: lookup table build with 'tmdb' ids
2023-01-02 09:11:38,409 DEBUG[PlexTraktSync]:Retry using search for specific Plex Episode tmdb://301771
2023-01-02 09:11:38,409 DEBUG[trakt.core]:get: https://api-v2launch.trakt.tv/search/tmdb/301771?type=episode
2023-01-02 09:11:38,409 DEBUG[trakt.core]:method, url :: get, https://api-v2launch.trakt.tv/search/tmdb/301771?type=episode
2023-01-02 09:11:38,411 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,411 DEBUG[trakt.core]:RESPONSE [get] (https://api-v2launch.trakt.tv/search/tmdb/301771?type=episode): request: GET https://api-v2launch.trakt.tv/search/tmdb/301771?type=episode, response: 200 (324 bytes), created: 2022-12-30 16:54:23 MST, expires: N/A (fresh)
2023-01-02 09:11:38,412 DEBUG[plexapi]:Reloading Episode 'Show 2' for attr 'guids'
2023-01-02 09:11:38,412 DEBUG[plexapi]:GET https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218952?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1
2023-01-02 09:11:38,413 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,414 DEBUG[requests_cache.policy.actions]:URL https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218952?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1 matched pattern "*/library/metadata/*?*include*": 0
2023-01-02 09:11:38,423 DEBUG[urllib3.connectionpool]:https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400 "GET /library/metadata/218952?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1 HTTP/1.1" 200 1238
2023-01-02 09:11:38,424 DEBUG[requests_cache.session]:Pre-cache checks for response from https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218952?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1: {'disabled cache': False, 'disabled method': False, 'disabled status': False, 'disabled by filter': False, 'disabled by headers or expiration params': True}
2023-01-02 09:11:38,425 DEBUG[requests_cache.session]:Skipping cache write for URL: https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/218952?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1
2023-01-02 09:11:38,428 DEBUG[PlexTraktSync]:Retry using search for specific Plex Episode tmdb://301777
2023-01-02 09:11:38,428 DEBUG[trakt.core]:get: https://api-v2launch.trakt.tv/search/tmdb/301777?type=episode
2023-01-02 09:11:38,428 DEBUG[trakt.core]:method, url :: get, https://api-v2launch.trakt.tv/search/tmdb/301777?type=episode
2023-01-02 09:11:38,429 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,430 DEBUG[trakt.core]:RESPONSE [get] (https://api-v2launch.trakt.tv/search/tmdb/301777?type=episode): request: GET https://api-v2launch.trakt.tv/search/tmdb/301777?type=episode, response: 200 (324 bytes), created: 2022-12-30 16:54:23 MST, expires: N/A (fresh)
2023-01-02 09:11:38,430 DEBUG[trakt.core]:get: https://api-v2launch.trakt.tv/search/tmdb/82596?type=show
2023-01-02 09:11:38,430 DEBUG[trakt.core]:method, url :: get, https://api-v2launch.trakt.tv/search/tmdb/82596?type=show
2023-01-02 09:11:38,431 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,431 DEBUG[trakt.core]:RESPONSE [get] (https://api-v2launch.trakt.tv/search/tmdb/82596?type=show): request: GET https://api-v2launch.trakt.tv/search/tmdb/82596?type=show, response: 200 (183 bytes), created: 2022-10-25 14:07:12 MDT, expires: N/A (fresh)
2023-01-02 09:11:38,431 DEBUG[plexapi]:GET https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364210/allLeaves
2023-01-02 09:11:38,432 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,432 DEBUG[requests_cache.policy.actions]:URL https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364210/allLeaves matched pattern "*/library/metadata/*/allLeaves": 0
2023-01-02 09:11:38,447 DEBUG[urllib3.connectionpool]:https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400 "GET /library/metadata/364210/allLeaves HTTP/1.1" 200 8026
2023-01-02 09:11:38,448 DEBUG[requests_cache.session]:Pre-cache checks for response from https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364210/allLeaves: {'disabled cache': False, 'disabled method': False, 'disabled status': False, 'disabled by filter': False, 'disabled by headers or expiration params': True}
2023-01-02 09:11:38,448 DEBUG[requests_cache.session]:Skipping cache write for URL: https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364210/allLeaves
2023-01-02 09:11:38,486 DEBUG[plexapi]:Reloading Episode 'Emily in Paris' for attr 'guids'
2023-01-02 09:11:38,486 DEBUG[plexapi]:GET https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364212?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1
2023-01-02 09:11:38,488 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,488 DEBUG[requests_cache.policy.actions]:URL https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364212?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1 matched pattern "*/library/metadata/*?*include*": 0
2023-01-02 09:11:38,506 DEBUG[urllib3.connectionpool]:https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400 "GET /library/metadata/364212?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1 HTTP/1.1" 200 3626
2023-01-02 09:11:38,507 DEBUG[requests_cache.session]:Pre-cache checks for response from https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364212?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1: {'disabled cache': False, 'disabled method': False, 'disabled status': False, 'disabled by filter': False, 'disabled by headers or expiration params': True}
2023-01-02 09:11:38,507 DEBUG[requests_cache.session]:Skipping cache write for URL: https://66-186-214-194.808cbc6741f14352b9c77bd45b676e74.plex.direct:32400/library/metadata/364212?checkFiles=1&includeAllConcerts=1&includeBandwidths=1&includeChapters=1&includeChildren=1&includeConcerts=1&includeExternalMedia=1&includeExtras=1&includeFields=thumbBlurHash%2CartBlurHash&includeGeolocation=1&includeLoudnessRamps=1&includeMarkers=1&includeOnDeck=1&includePopularLeaves=1&includePreferences=1&includeRelated=1&includeRelatedCount=1&includeReviews=1&includeStations=1
2023-01-02 09:11:38,515 DEBUG[trakt.core]:get: https://api-v2launch.trakt.tv/shows/148191/seasons?extended=episodes
2023-01-02 09:11:38,515 DEBUG[trakt.core]:method, url :: get, https://api-v2launch.trakt.tv/shows/148191/seasons?extended=episodes
2023-01-02 09:11:38,516 DEBUG[requests_cache.policy.actions]:Cache directives from request headers: {}
2023-01-02 09:11:38,516 DEBUG[requests_cache.policy.actions]:URL https://api-v2launch.trakt.tv/shows/148191/seasons?extended=episodes matched pattern "*.trakt.tv/shows/*/seasons": 0
2023-01-02 09:11:38,551 DEBUG[urllib3.connectionpool]:https://api-v2launch.trakt.tv:443 "GET /shows/148191/seasons?extended=episodes HTTP/1.1" 200 None
2023-01-02 09:11:38,552 DEBUG[requests_cache.session]:Pre-cache checks for response from https://api-v2launch.trakt.tv/shows/148191/seasons?extended=episodes: {'disabled cache': False, 'disabled method': False, 'disabled status': False, 'disabled by filter': False, 'disabled by headers or expiration params': True}
2023-01-02 09:11:38,553 DEBUG[requests_cache.session]:Skipping cache write for URL: https://api-v2launch.trakt.tv/shows/148191/seasons?extended=episodes
2023-01-02 09:11:38,554 DEBUG[trakt.core]:RESPONSE [get] (https://api-v2launch.trakt.tv/shows/148191/seasons?extended=episodes): <Response [200]>
2023-01-02 09:11:38,554 DEBUG[plexapi]:Reloading Episode 'Masculin Féminin' for attr 'guids'
```

### Expected behavior

In the attached log, for example, he had not watched any of "Emily in Paris," and we ensured the show was unwatched in both Plex and Trakt before this sync occurred. The expected behavior, of course, was that it would not be added to his watch history.

### Steps to reproduce the behavior

1. Run ~/.local/pipx/venvs/plextraktsync/bin/plextraktsync (I have this set to run every 30 mins, using Lingon)
2. I'll periodically notice episodes have been marked as watched, even though they haven't
3. Mark each episode as unwatched in Plex and Trakt
4. Wait, hoping this won't happen again (it usually does within a few days)

### Workarounds

Every time this happens, our solution has been manually marking these episodes as unwatched on both Plex and Trakt.

So far, we have also tried setting up his instance of PTS from scratch, ensuring both Plex and Trakt are logged in as him, and that his own credentials were used in the setup. 

### Install method

pipx

### Config file contents

```yaml
cache:
  path: $PTS_CACHE_DIR/trakt_cache

excluded-libraries:
  - Clips
  - Fitness
  - Music
  - Music Videos
  - Wrestling

config:
  dotenv_override: true

plex:
  timeout: 90

logging:
  append: true
  # Whether to show timestamps in console messages
  console_time: true
  debug: true
  filename: plextraktsync.log

sync:
  plex_to_trakt:
    collection: true
    ratings: true
    watched_status: true
    watchlist: true
  trakt_to_plex:
    liked_lists: false
    ratings: true
    watched_status: true
    watchlist: false
    watchlist_as_playlist: false

watch:
  add_collection: false
  remove_collection: false
  scrobble_threshold: 80
  username_filter: true

xbmc-providers:
  movies: imdb
  shows: tvdb

# vim:ts=2:sw=2:et
```

### Version

0.25.3

### Python Version

3.10.2

### Operating System and Version

macOS 13.1

**Comment by glensc** [comment]
so, are you running only sync command, or also watch command? also to clarify be clear you both run plextraktsync, you are owner but the other user is not and both use same plex server? and the log you shared is your user or the other user? maybe show as diagram  or table somehow? :D

**Comment by pterisaur** [comment]
We are running plextraktsync fully (I don't specify sync) every 30 minutes. Our two instances are set up on different computers, connecting to different Plex accounts on the same server and different Trakt accounts. 

The log is from my partner's instance. Both our configs look the same. 

How do I get an inspect command?

**Comment by glensc** [comment]
there's no "fully sync", plextraktsync without argument will runs sync command, in fact, it will print you a deprecated message about that. inspect is subcommand of plextraktsync, use --help to see commands.

you didn't clarify who has admin access with their plex account to the plex server.

output of inspect of wrongly synced id is needed from both accounts without manually fixing play state.

**Comment by glensc** [comment]
Id in your log was 218949

```
$ plextraktsync inspect 218949
```

but you need to do that for id that's broken, doing it now won't I've any use info

**Comment by pterisaur** [comment]
Ahh, ok so I found the deprecated warning at the beginning of the log, so we've been running sync without argument. We'll have to change that.

I am the Plex server owner and admin. My stuff has been syncing just fine — it's my partner's instance that has been behaving badly.

As far as IDs go, I'm still not sure I understand what you're asking. I tried "inspect" and it only told me the app version number. I tried doing the watched-shows table, and I do see IDs listed with shows that should not be marked watched:

39160 smash
77677 marvel-s-agent-carter
158804 halston

We've marked these as unwatched in Plex and Trakt — is there something more we can do to ensure they don't get marked as watched again?
